docs/source/changelog.md

```diff
@@ -1,16 +1,14 @@
-What's new
-==========
+# What\'s new
 
-poliastro 0.14.0 - 2020-05-08
------------------------------
+## poliastro 0.14.0 - 2020-05-08
 
 This major release contains crucial new features and bug fixes that have
-been years in the making, and is by far the most exciting release
-in the history of the project.
+been years in the making, and is by far the most exciting release in the
+history of the project.
 
-Highlights
-..........
+### Highlights
 
+```{eval-rst}
```

Comment:
What part of these `{eval-rst}` blocks do not work in normal MyST?

Reply:
Actually, to make the extension such as `hoverxref` work, they need the `:py:class` / `:py:nature_of_the_term_in_consideration` tag [Some other need, some other tinkering to work which is native to the `.rst` syntax]. 
MyST doesn't support inline roles for these tags & hence we need to keep the whole block under this `{eval-rst}` directive. Also, `{eval-rst}` whenever parsed creates a blank line above & below the part of the code that is kept inside it, so the formatting of many paragraphs is hindered. I made the changes so that these random formatting can be minimised :)